lib/resty/worker/events.lua

```diff
@@ -272,15 +273,14 @@ _M.post = function(source, event, data, unique)
     err = 'failed posting event "'..event..'" by "'..
           source..'"; '..tostring(err)
     log(ERR, "worker-events: ", err)
-    return success, err
   end
 
-  return _M.poll()
+  return true, err
 end
 
 -- the same as post. But the event will only be handled in the worker
 -- it was posted from, it will not be broadcasted to other worker processes.
--- @return results from the call to `poll`, or nil+error
+-- @return `true` or nil+error
```

Comment:
I think we should rewrite this:
```
-- the same as post. But the event will only be handled in the worker
-- it was posted from, it will not be broadcasted to other worker processes.
```
to
```
-- Similar to `post`, but synchronous. The event will be immediately executed
-- and will not require a call to `poll`. It will only be handled in the worker
-- it was posted from, it will not be broadcasted to other worker processes.
```
@hishamhm what do you think about this interface? since `post` and `post_local` now deviate in sync/async behaviour. Though each of them in themselves are no longer mixing those behaviours, so that's good I guess.

Reply:
It's really clearer that way.